bitrepository-integrity-service/src/main/java/org/bitrepository/integrityservice/workflow/SaltedChecksumWorkflow.java

```diff
@@ -131,7 +130,11 @@ private ChecksumSpecTYPE getChecksumSpecWithRandomSalt() {
         }
 
         String salt = UUID.randomUUID().toString().replace("-", "");
-        res.setChecksumSalt(Base16Utils.encodeBase16(salt));
+        try {
+            res.setChecksumSalt(Base16Utils.encodeBase16(salt));
+        } catch (DecoderException e) {
+            log.error(e.getMessage());
```

Comment:
Execution would again not stop here as it would before. Doesn't really matter though as 'salt' should always be a valid hex string here.